.github/workflows/test_security.yml

```diff
@@ -72,11 +72,11 @@ jobs:
       - name: Run AD Test
         if: env.imagePresent == 'true'
         run: |
-          security=`curl -XGET https://localhost:9200/_cat/plugins?v -u admin:admin --insecure |grep opensearch-security|wc -l`
+          security=`curl -XGET https://localhost:9200/_cat/plugins?v -u admin:myStrongPassword123! --insecure |grep opensearch-security|wc -l`
```

Comment:
this password needs to be passed to the containers on line 70 with `-e OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123!`

There is no default password for admin user starting 2.12. This password needs to be set, and you can set it as above.

Also, this change should only be present for branches pointing to 2.12 image.

Reply:
> Also, this change should only be present for branches pointing to 2.12 image.

Should this change be present for branches pointing to 2.12 image or 2.12 and above image?

Reply:
The change should be present starting from 2.12. (not in 2.11.1 and below).

> This change is towards main branch and will be backported to 2.x branch

yes.